application/backup.sh

```diff
@@ -1,114 +1,133 @@
+        log "ERROR" "mysql-backup-restore: FATAL: Backup of ${dbName} returned non-zero status ($STATUS) in $(expr ${end} - ${start}) seconds."
         exit $STATUS
     else 
-        echo "mysql-backup-restore: Backup of ${dbName} completed in $(expr ${end} - ${start}) seconds, ($(stat -c %s /tmp/${dbName}.sql) bytes)."
+        log "INFO" "mysql-backup-restore: Backup of ${dbName} completed in $(expr ${end} - ${start}) seconds, ($(stat -c %s /tmp/${dbName}.sql) bytes)."
     fi
 
+    # Compression
     start=$(date +%s)
     gzip -f /tmp/${dbName}.sql
     STATUS=$?
     end=$(date +%s)
     if [ $STATUS -ne 0 ]; then
         error_message="Compression failed for database ${dbName} backup"
         error_to_sentry "$error_message" "$dbName" "$STATUS"
-        echo "mysql-backup-restore: FATAL: Compressing backup of ${dbName} returned non-zero status ($STATUS) in $(expr ${end} - ${start}) seconds."
+        log "ERROR" "mysql-backup-restore: FATAL: Compressing backup of ${dbName} returned non-zero status ($STATUS) in $(expr ${end} - ${start}) seconds."
         exit $STATUS 
     else
-        echo "mysql-backup-restore: Compressing backup of ${dbName} completed in $(expr ${end} - ${start}) seconds."
+        log "INFO" "mysql-backup-restore: Compressing backup of ${dbName} completed in $(expr ${end} - ${start}) seconds."
     fi
 
+    # S3 Upload
     start=$(date +%s)
     s3cmd put /tmp/${dbName}.sql.gz ${S3_BUCKET}
     STATUS=$?
     end=$(date +%s)
     if [ $STATUS -ne 0 ]; then
         error_message="S3 copy failed for database ${dbName} backup"
         error_to_sentry "$error_message" "$dbName" "$STATUS"
-        echo "mysql-backup-restore: FATAL: Copy backup to ${S3_BUCKET} of ${dbName} returned non-zero status ($STATUS) in $(expr ${end} - ${start}) seconds."
+        log "ERROR" "mysql-backup-restore: FATAL: Copy backup to ${S3_BUCKET} of ${dbName} returned non-zero status ($STATUS) in $(expr ${end} - ${start}) seconds."
         exit $STATUS
     else
-        echo "mysql-backup-restore: Copy backup to ${S3_BUCKET} of ${dbName} completed in $(expr ${end} - ${start}) seconds."
+        log "INFO" "mysql-backup-restore: Copy backup to ${S3_BUCKET} of ${dbName} completed in $(expr ${end} - ${start}) seconds."
     fi
 
+    # Backblaze B2 Upload (Optional)
     if [ "${B2_BUCKET}" != "" ]; then
         start=$(date +%s)
         s3cmd \
-            --access_key=${B2_APPLICATION_KEY_ID} \
-	    --secret_key=${B2_APPLICATION_KEY} \
-	    --host=${B2_HOST} \
-	    --host-bucket='%(bucket)s.'"${B2_HOST}" \
-	    put /tmp/${dbName}.sql.gz s3://${B2_BUCKET}/${dbName}.sql.gz
+        --access_key=${B2_APPLICATION_KEY_ID} \
+        --secret_key=${B2_APPLICATION_KEY} \
+        --host=${B2_HOST} \
+        --host-bucket='%(bucket)s.'"${B2_HOST}" \
+        put /tmp/${dbName}.sql.gz s3://${B2_BUCKET}/${dbName}.sql.gz
```

Comment:
Leave this indentation since it's all part of the `s3cmd` command line.
```suggestion
            --access_key=${B2_APPLICATION_KEY_ID} \
            --secret_key=${B2_APPLICATION_KEY} \
            --host=${B2_HOST} \
            --host-bucket='%(bucket)s.'"${B2_HOST}" \
            put /tmp/${dbName}.sql.gz s3://${B2_BUCKET}/${dbName}.sql.gz
```